baby-gru/src/types/libcoot.d.ts

```diff
@@ -467,7 +468,9 @@ export namespace libcootApi {
         moved_atom_t: { new(arg0: string, arg1: string, arg2: number, arg3: number, arg4: number, arg5: number): MovedAtomT};
         MapIntFloat3: { new(): emscriptem.map<[number, number, number], number>};
         VectorStringUInt_pair: { new(): emscriptem.vector<{ first: string, second: number }>};
-        getRamachandranData(arg0: string, arg1: string): emscriptem.vector<RamaData>
+        getRamachandranData(arg0: string, arg1: string): emscriptem.vector<RamaData>;
+
+        validate(arg0: string, arg1: string): emscriptem.vector<any>
```

Comment:
The return type here should be `emscriptem.vector<privateer.PrivateerResultsEntry>`